engine/security/certificates.md

```diff
@@ -23,12 +23,12 @@ A custom certificate is configured by creating a directory under
 `/etc/docker/certs.d` using the same name as the registry's hostname, such as
 `localhost`. All `*.crt` files are added to this directory as CA roots.
 
-> **Note**:
-> As of Docker 1.13, on Linux any root certificates authorities are merged
-> with the system defaults, including as the host's root CA set. On prior
-versions of Docker, and on Docker Enterprise Edition for Windows Server,
-> the system default certificates are only used when no custom root certificates
-> are configured.
+> **Note**
+>
+> On Linux any root certificates authorities are merged with the system defaults,
+> including as the host's root CA set. On Docker Enterprise Edition for Windows
```

Comment:
Should we also remove the Docker Enterprise Edition here?

Reply:
Ah, yes, I was looking at that. It was a bit tricky as it applies to either "Docker on Windows" or "Docker Desktop on Windows running "Windows containers". Let me have another look